lib/platform/bitbucket-server/index.js

```diff
@@ -651,6 +658,25 @@ async function createPr(
 ) {
   logger.debug(`createPr(${branchName}, title=${title})`);
   const base = useDefaultBranch ? config.defaultBranch : config.baseBranch;
+  let reviewers = [];
+
+  /* istanbul ignore else */
+  if (config.bbUseDefaultReviewers) {
```

Comment:
It was my intention that we should first check if there are reviewers sent in the createPr and if so then use those. Then if not, check for default reviewers.

Reply:
just to remove the istanbul info, that the else path is not taken.

Reply:
the reviewers from config are not send to createPr, so i can't check